src/main/java/com/learning/pages/google/GoogleSearchPage.java

```diff
@@ -1,26 +1,27 @@
 package com.learning.pages.google;
 
-import com.learning.configuration.PropertiesReader;
+import com.learning.browsers.Browser;
 import com.learning.pages.PageBase;
 import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.ui.WebDriverWait;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
 
+@Component
 public class GoogleSearchPage extends PageBase {
 
+  private final String googleURL;
```

Comment:
URL is not a property of PO, it's Browser state, thus it should not be available via PO method.

Reply:
As discussed it can stay.